src/runtime/internal/utils.ts

```diff
@@ -32,6 +33,29 @@ export function run_all(fns) {
 	fns.forEach(run);
 }
 
+// export function error_capture(){
+// 	const onerror = window.onerror || noop;
+// 	window.onerror = function(e) {
+// 		error_handle(e);
+// 		onerror(e);
+// 	};
+// }
+
+export function error_handler(fn) {
+	try { 
```

Comment:
Improvement: check if any subscribers iand if we should generate try-catch block